src/rust/engine/src/lib.rs

```diff
@@ -546,6 +547,23 @@ pub extern fn rule_subgraph_visualize(
   })
 }
 
+#[no_mangle]
+pub extern fn set_panic_handler() {
+  panic::set_hook(Box::new(|panic_info| {
+    let mut panic_str = format!("panic at '{}', ",
+                                panic_info.payload().downcast_ref::<&str>().unwrap());
+
+    if let Some(location) = panic_info.location() {
+      let panic_location_str = format!("{}:{}", location.file(), location.line());
+      panic_str.push_str(&panic_location_str);
```

Comment:
should probably move the inclusion of the trailing comma here to the later conditional `push_str`'d string?

if the `panic_info.location()` `Option` is `None`, some logs could look like: `panic at '...', ` which might make them appear to be truncated.